### DISCUSSION TITLE
Where to find Gene expression data file for mouse and beagle dogs

### DISCUSSION THREAD

**Discussion by Gautamvj** [Open-Systems-Pharmacology/Forum#1244] [Models]
Hello Community folks,
Can anyone guide me where to find the gene expression data file mouse and beagle dogs?

Regards,
Gautam Vijaywargi.

**Comment by StephanSchaller** [comment]
Hi, these are just a prototype / beta version so far. See the posts here:
(rat and dog already made available, mouse / beagle probably once the publication is out)
https://github.com/Open-Systems-Pharmacology/Forum/discussions/1096

**Reply by Gautamvj** [reply]
Thank you Stephan for this. Found the file for Dog and looking forward for the mouse GDB file and also the publication.
